### PR TITLE
Add `Verified users`

### PR DESCRIPTION
**Pull request by BlueChinchompa**

### CODE REVIEW

File: wiki/People/Verified_Users/en.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Verified Users
+
+Verified Users are well-known/top players who have verified their identity.
```
Comment (iSlodinxOsu) [inline wiki/People/Verified_Users/en.md:3]:
```suggestion
Verified users are top players or content creators who have verified their identity.
```

File: wiki/People/Verified_Users/en.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Verified Users
```
Comment (iSlodinxOsu) [inline wiki/People/Verified_Users/en.md:1]:
```suggestion
# Verified users
```

File: wiki/People/Verified_Users/en.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+Verified Users are well-known/top players who have verified their identity.
+
+
+## Current Verified Users
```
Comment (iSlodinxOsu) [inline wiki/People/Verified_Users/en.md:6]:
```suggestion
## Current Verified users
```

File: wiki/People/Verified_Users/en.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+User | Bancho Account
+---|:---:
+[TheShadowOfDark](https://osu.titanic.sh/u/64)  | [TheShadowOfDark](https://osu.ppy.sh/users/5795337)
```
Comment (iSlodinxOsu) [inline wiki/People/Verified_Users/en.md:10]:
```suggestion
[TheShadowOfDark](https://osu.titanic.sh/u/64) | [TheShadowOfDark](https://osu.ppy.sh/users/5795337)
```

File: wiki/People/Verified_Users/en.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+[index](https://osu.titanic.sh/u/82)            | [Dawns](https://osu.ppy.sh/users/4639477)
```
Comment (iSlodinxOsu) [inline wiki/People/Verified_Users/en.md:11]:
```suggestion
[index](https://osu.titanic.sh/u/82) | [Dawns](https://osu.ppy.sh/users/4639477)
```

File: wiki/People/Verified_Users/en.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+[Spare](https://osu.titanic.sh/u/92)            | [Spare](https://osu.ppy.sh/users/2204373)
```
Comment (iSlodinxOsu) [inline wiki/People/Verified_Users/en.md:12]:
```suggestion
[Spare](https://osu.titanic.sh/u/92) | [Spare](https://osu.ppy.sh/users/2204373)
```

File: wiki/People/Verified_Users/en.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+[Champ](https://osu.titanic.sh/u/96)            | [The Champion](https://osu.ppy.sh/users/1719471)
```
Comment (iSlodinxOsu) [inline wiki/People/Verified_Users/en.md:13]:
```suggestion
[Champ](https://osu.titanic.sh/u/96) | [The Champion](https://osu.ppy.sh/users/1719471)
```

File: wiki/People/Verified_Users/en.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+[Plasma](https://osu.titanic.sh/u/191)          | [Plasma](https://osu.ppy.sh/users/10077431)
```
Comment (iSlodinxOsu) [inline wiki/People/Verified_Users/en.md:14]:
```suggestion
[Plasma](https://osu.titanic.sh/u/191) | [Plasma](https://osu.ppy.sh/users/10077431)
```

File: wiki/People/Verified_Users/en.md [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+[mrekk](https://osu.titanic.sh/u/208)           | [mrekk](https://osu.ppy.sh/users/7562902)
```
Comment (iSlodinxOsu) [inline wiki/People/Verified_Users/en.md:15]:
```suggestion
[mrekk](https://osu.titanic.sh/u/208) | [mrekk](https://osu.ppy.sh/users/7562902)
```

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]
Extra spaces are not necessary to add, once Wiki is released, we can see how it looks like and then make another PR after that (if needed).

**Review by iSlodinxOsu — APPROVED** [review]

**Review by Meruru-chan — APPROVED** [review]
I'm not a verified user it's over...

**Pull request merged**